test/truffle-fixture.js

```diff
@@ -124,6 +138,26 @@ async function deployContracts() {
 
   const reputationMiningCycleBinarySearch = await ReputationMiningCycleBinarySearch.new();
   ReputationMiningCycleBinarySearch.setAsDeployed(reputationMiningCycleBinarySearch);
+
+  // Deploy CreateX if it's not already deployed
+  const createXCode = await web3.eth.getCode(CREATEX_ADDRESS);
+  if (createXCode === "0x") {
+    const accounts = await web3.eth.getAccounts();
+    await web3.eth.sendTransaction({
+      from: accounts[0],
+      to: "0xeD456e05CaAb11d66C4c797dD6c1D6f9A7F352b5",
+      value: web3.utils.toWei("0.3", "ether"),
+      gasPrice: web3.utils.toWei("1", "gwei"),
+      gas: 300000,
+    });
+    const rawTx = fs
+      .readFileSync("lib/createx/scripts/presigned-createx-deployment-transactions/signed_serialised_transaction_gaslimit_3000000_.json", {
+        encoding: "utf8",
+      })
+      .replace(/"/g, "")
+      .replace("\n", "");
+    await web3.eth.sendSignedTransaction(rawTx);
```

Comment:
I'd prefer all of this be captured in some utility function somewhere, perhaps in `utils/createx.js` or something along those lines.